app/models/departureP5/FunctionalError.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.departureP5
+
+import play.api.libs.json.{Json, Reads}
+
+case class FunctionalError(
+  errorPointer: String,
+  errorCode: String,
+  errorReason: String,
+  originalAttributeValue: Option[String]
+) {
+
+  def isAmendable: Boolean = errorPointer.startsWith("/CC015C/")
```

Comment:
I don't think this makes it amendable as such? That is the parent that will contain the relevant sections. So for something to be amendable it needs to have a section inside CC015C that we can link back to a section in our FE.